### PR TITLE
Astro 8/31 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODA0NzE

### CODE REVIEW

File: astro/deployment-metrics.md [hunk 1]
```diff
@@ -177,6 +177,16 @@ To access the **DAGs** page, you can either click the DAGs icon in the UI or cli
 
 ![DAGs page](/img/docs/dags-page.png)
 
+## Export Airflow metrics to Datadog
+
+You can export metrics about a Deployment's Airflow environment to Datadog. The Datadog Agent collects metrics that are available in the Astro UI as well as more specific metrics about your environment's performance. See the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) for a complete list of exported metrics.
```
Comment (kwcanuck) [inline astro/deployment-metrics.md:182]:
```suggestion
You can now export Airflow Deployment metrics to Datadog. The Datadog Agent collects metrics that are available in the Astro UI and environment performance metrics. See the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) for a complete list of exported metrics.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,22 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.4.0 ([Release notes](cli/release-notes.md))
 
+## August 31, 2022
+
+### Export Airflow metrics to Datadog
+
+You can now export metrics about a Deployment's Airflow environment to Datadog through an Astronomer-supported integration. After you configure this feature, your Deployment exports all available metrics to Datadog as described in the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected).
```
Comment (kwcanuck) [inline astro/release-notes.md:22]: See previous edit recommendations.

Comment (kwcanuck) [inline astro/release-notes.md:22]:
```suggestion
An Astronomer-supported integration now allows you to export Airflow Deployment metrics to Datadog. After you configure this feature, your Deployment exports all available metrics to Datadog. For more information about the type of metrics you can export, see the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected).
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,22 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.4.0 ([Release notes](cli/release-notes.md))
 
+## August 31, 2022
+
+### Export Airflow metrics to Datadog
```
Comment (kwcanuck) [inline astro/release-notes.md:20]:
```suggestion
### Export Airflow Deployment metrics to Datadog
```

**Review by kwcanuck — APPROVED** [review]

File: astro/deployment-metrics.md [hunk 4]
```diff
@@ -177,6 +177,16 @@ To access the **DAGs** page, you can either click the DAGs icon in the UI or cli
...
+
+1. Create a new Datadog API key or copy an existing API key. See [Datadog documentation](https://docs.datadoghq.com/account_management/api-app-keys/).
+2. [Create a new environment variable](environment-variable.md) in your Deployment. Specify the variable key as `DATADOG_API_KEY` and its value as your Datadog API key. Select the **Secret** checkbox.
+3. Save the environment variable.
+
+After you configure the environment variable, Astro launches a sidecar container in your Deployment that runs [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent). This container works with your Deployment's existing StatsD infrastructure to export Airflow metrics to the Datadog instance associated with your API key.
```
[the remark this answers is not in the document]
Comment (howardyoo) [reply astro/deployment-metrics.md:188]: Please add a note at the end of this section, specifying that this integration will NOT be collecting `service checks` on the Airflow instance (https://docs.datadoghq.com/integrations/airflow/?tab=host#service-checks), so users won't be able to see them.

**Comment by chomipi88** [comment]
I would also like to draw attention to https://app.zenhub.com/workspaces/managed-airflow---admin-625732835c23b0001411e270/issues/astronomer/control-plane-auth-proxy/122

This has been a painful issue for users stuck accessing Airflow UI requiring them to clear all cookies and cache for *.astronomer.run and *.astronomer.io domains to be able to log in again.
We are not handling that on the user's behalf and will return them to the login form if the tokens are invalid or expired.

**Review by howardyoo — COMMENTED** [review]
LGTM! Thank you!

File: astro/deployment-metrics.md (hunk 1 above)
Comment (jwitz) [reply astro/deployment-metrics.md:182]: This edit makes the feature sound automatic, when the exporting must be manually configured

**Review by chomipi88 — COMMENTED** [review]
see comment

**Comment by chomipi88** [comment]
> > I would also like to draw attention to https://app.zenhub.com/workspaces/managed-airflow---admin-625732835c23b0001411e270/issues/astronomer/control-plane-auth-proxy/122
> > This has been a painful issue for users stuck accessing Airflow UI requiring them to clear all cookies and cache for *.astronomer.run and *.astronomer.io domains to be able to log in again. We are not handling that on the user's behalf and will return them to the login form if the tokens are invalid or expired.
> 
> @chomipi88 , I believe the above comment was intended for different issue.

The ask was to include in release notes

**Comment by jwitz** [comment]
@chomipi88 hotfixed the release notes after these were merged, there's now a bug fix note for the change you linked!

**Pull request merged**